classes/ValidFormBuilder/Area.php

```diff
@@ -552,7 +577,7 @@ public function getValue($intCount = null)
      */
     public function getId()
     {
-        return $this->getName() . "_wrapper";
+        return $this->getName() . "_vf_wrapper";
```

Comment:
I could undo that. But adding `_wrapper` is a bit too generic. 

That being said it has nothing to do with this PR so I'll get rid of it.